src/dune_pkg/solver_env.ml

```diff
@@ -224,19 +224,26 @@ let default =
   { flags = Variable.Flag.Set.all
   ; sys = Variable.Sys.Bindings.empty
   ; const = Variable.Const.bindings
-  ; repos = [ Workspace.Repository.Name.of_string ":standard" ] (* TODO *)
+  ; repos = [ Workspace.Repository.Name.of_string ":standard" ]
```

Comment:
How can the name of a repo be `:standard`? I imagine this should be something like `default`? The same way we have a "default" context.

Reply:
Shouldn't it be the ":standard" set of repositories rather than a repo itself?

Reply:
It's because the name of a repository is just a string and I predefined a repository that's called `:standard` (that's how it worked before the Ordered set lang so if someone specified `:standard` it would pick just that). I can rename it into `default` now that we are using Ordered set lang.

Reply:
> Shouldn't it be the ":standard" set of repositories rather than a repo itself?

This is the ordered set of desired repos, the joining with the list of repos comes later (especially since this is `solver_env`. `repositories` lives in a different stanza I don't think joining it here makes it more readable).

Reply:
Renamed the repo to `default`.